### PR TITLE
[greynoise-266] New Pack - FeedGreyNoiseIndicator

### PR DESCRIPTION
**Pull request by bradchiappetta**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

Adds a new pack support using GreyNoise as an Indicator Feed.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bradchiappetta** [comment]
@shmuel44 : I believe this is ready for your review.  Open to all comments and ways to improve this to meet your standards.

**Comment by shmuel44** [comment]
Hi @bradchiappetta ,
Thank you for your contribution!
Good work :)

Would it be worth considering adding the `GreyNoiseIndicator` as a new integration to the `GreyNoise` pack? 
I'm curious if there's a specific reason why we would need to create a new pack.

**Comment by bradchiappetta** [comment]
> Hi @bradchiappetta , Thank you for your contribution! Good work :)
> 
> Would it be worth considering adding the `GreyNoiseIndicator` as a new integration to the `GreyNoise` pack? I'm curious if there's a specific reason why we would need to create a new pack.

I was told since it was a Feed integration it had to be it's own pack? Happy to do whatever y'all prefer.

File: Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/GreyNoiseIndicator.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  type: 8
+  required: false
+description: "GreyNoise is a cybersecurity platform that collects and analyzes Internet-wide scan and attack traffic. With this integration, users can contextualize existing alerts, filter false-positives, identify compromised devices, and track emerging threats. This Integration provides a feed of IPv4 Internet Scanners from GreyNoise."
+display: GreyNoise Indicator Feed
+name: GreyNoise Indicator Feed
+script:
+  commands:
+  - arguments: []
+    description: Gets the feed indicators.
+    name: greynoise-get-indicators
```
Comment (shmuel44) [inline Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/GreyNoiseIndicator.yml:107]: Do you know what the outputs of the command are?
For more details on outputs, please refer to https://xsoar.pan.dev/docs/integrations/context-and-outputs.

Comment (shmuel44) [inline Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/GreyNoiseIndicator.yml:107]:
```suggestion
  - description: Gets the feed indicators.
    name: greynoise-get-indicators
```

File: Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/README.md
Comment (shmuel44) [inline Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/README.md:1]: Please use the command `demisto-sdk generate-docs`. 
 For more detailed information, please refer to https://xsoar.pan.dev/docs/documentation/readme_file#creating-documentation.

**Review by shmuel44 — COMMENTED** [review]
Good Work!

File: Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/GreyNoiseIndicator.yml (hunk 1 above)
Comment (bradchiappetta) [reply Packs/FeedGreyNoiseIndicator/Integrations/GreyNoiseIndicator/GreyNoiseIndicator.yml:106]: So, this command generates what I think is the standard table for an indicator feed in the war room, and those indicators get added to the TIM.  I'm not sure what outputs would be captured here.  I could use some guidance on this one.

**Review by MLainer1 — COMMENTED** [review]
TIM review complete 🚀

**Comment by bradchiappetta** [comment]
@shmuel44 @MLainer1 I see a bunch of CI failures all of the sudden, but I don't think they are related to my PR?  Anything else needed from me at this point to get this finished?

**Review by shmuel44 — APPROVED** [review]

**Comment by shmuel44** [comment]
Hi @bradchiappetta,
I merged the internal PR, and your update is now in the master branch of the content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Pull request merged**